core/modules/admin_bar/css/admin_bar.css

```diff
@@ -68,14 +68,142 @@
+  padding-left: 29px; /* LTR */
+}
+[dir="rtl"] #admin-bar .dropdown li.admin-bar-icon > a {
+  background-position: 95% center;
+  padding-left: 9px;
+  padding-right: 29px;
+}
+
+/* Admin bar label */
+#admin-bar #admin-bar-menu.dropdown li > span.menu {
+  background: transparent url(../images/bars--white--64.png) 5% center no-repeat; /* LTR */
+  padding-left: 27px; /* LTR */
+  background-size: 16px;
+}
+[dir="rtl"] #admin-bar #admin-bar-menu.dropdown li > span.menu {
+  background-position: 95% center;
+  padding-left: 9px;
+  padding-right: 27px;
+}
+
+/* All admin bar menu links */
+#admin-bar #admin-bar-menu > li > .dropdown > li > a {
+  padding-left: 29px; /* LTR */
+}
+[dir="rtl"] #admin-bar #admin-bar-menu > li > .dropdown > li > a {
+  padding-left: 9px;
+  padding-right: 29px;
+}
+
+/* Specific admin bar menu links */
+#admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-content {
+  background: transparent url(../images/pencil--white--64.png) 5% center no-repeat;
+  background-size: 16px;
+}
+#admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-people {
+  background: transparent url(../images/users--white--64.png) 5% center no-repeat;
+  background-size: 16px;
+}
+#admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-appearance {
+  background: transparent url(../images/palette--white--64.png) 5% center no-repeat;
+  background-size: 16px;
+}
+#admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-modules {
+  background: transparent url(../images/puzzle-piece--white--64.png) 5% center no-repeat;
+  background-size: 16px;
+}
+#admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-structure {
+  background: transparent url(../images/sort-amount-down--white--64.png) 5% center no-repeat;
+  background-size: 16px;
+}
+#admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-config {
+  background: transparent url(../images/cog--white--64.png) 5% center no-repeat;
+  background-size: 16px;
+}
+#admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-devel {
+  background: transparent url(../images/code--white--64.png) 5% center no-repeat;
+  background-size: 16px;
+}
+#admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-reports {
+  background: transparent url(../images/info-circle--white--64.png) 5% center no-repeat;
+  background-size: 16px;
+}
+[dir="rtl"] #admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-content,
+[dir="rtl"] #admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-people,
+[dir="rtl"] #admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-appearance,
+[dir="rtl"] #admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-modules,
+[dir="rtl"] #admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-structure,
+[dir="rtl"] #admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-config,
+[dir="rtl"] #admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-devel,
+[dir="rtl"] #admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-reports {
```

Comment:
...wondering whether we could replace all these with a single `[dir="rtl"] #admin-bar #admin-bar-menu > li > .dropdown > li > a`

Reply:
I didn't think that would work due to selector specificity, but it totally did! 
<img width="1372" alt="screen shot 2019-01-11 at 3 57 30 pm" src="https://user-images.githubusercontent.com/397895/51065436-a56c9300-15b9-11e9-81da-09d0de7e710f.png">